src/pages/seoyeongYoon/Main/MainSeo.js

```diff
@@ -1,7 +1,200 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import '../../seoyeongYoon/Main/MainSeo.scss';
```

Comment:
```suggestion
import './MainSeo.scss';
```

scss 파일이 같은 폴더 안에 있는 경우
위와 같이 경로를 지정할 수 있습니다!